test/verification.test.js

```diff
@@ -180,5 +180,8 @@ test("fetchVerificationKeys()", async (t) => {
     request: testRequest,
   });
 
-  t.deepEqual(result, publicKeys);
+  t.deepEqual(result, { cacheId: "", keys: publicKeys });
 });
+
+// Up next: test the cache
```

Comment:
⏸️ Paused here to mark progress. I don't (yet) understand how `MockAgent` works, but I'll see what a good night's sleep will do and pick this back up tomorrow or Friday. ✌️

Reply:
I'm still confused by the multiple layers of network requests by undici's `fetch`. No need to understand all the inner wokrings, you should be fine copying from other tests.